src/macros.rst

```diff
@@ -149,38 +150,46 @@ A :t:`fragment specifier` is a :t:`construct` that indicates the :t:`type` of
 a :t:`metavariable`.
 
 :dp:`fls_8o9mcV2KrKac`
-:t:`[metavariable]s` have :t:`[fragment specifier restriction]s` on what may follow them in a :s:`MacroMatcher`:
+:t:`Fragment specifier` kinds impose the following
+:dt:`[fragment specifier restruction]s` on the :t:`[token]s` that follow them:
```

Comment:
restruction => restriction

Specify somewhere that these restrictions are on the call site, not the macro definition. The example below contains no macro calls.

Reply:
Thanks for catching this, will fix!

Reply:
@riking
@Veykril indicated that the current phrasing is actually correct, the restrictions apply to the [macro declaration](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=42e223f70885d65d3a3e8b63b980b730), not the macro invocation.